playwright-dependencies/tests/smoke.spec.ts

```diff
@@ -0,0 +1,42 @@
+import { test } from '@playwright/test';
+
+test('smokeTest', async ({ page }) => {
+  const url = "localhost:" + process.env.CONTAINER_EXPORTED_PORT;
+
+  await page.goto(url);
+
+  await page.getByRole('button', { name: 'Sign In' }).click();
+  await page.getByLabel('Email Address').click();
+  await page.getByLabel('Email Address').fill('test@liferay.com');
+  await page.getByLabel('Email Address').press('Tab');
+  await page.getByLabel('Password').fill('test');
+  await page.getByLabel('Password').press('Tab');
+  await page.getByLabel('Remember Me').press('Tab');
+  await page.getByLabel('Sign In- Loading').getByRole('button', { name: 'Sign In' }).press('Enter');
+
+  await page.locator(".float-right").waitFor({state: "visible"});
```

Comment:
@lucasmiranda0, isn't this locator too generic? Is there any chance of we make it more specific to prevent undesired behaviors?